package.json

```diff
@@ -4,19 +4,32 @@
   "description": "A smart request-making library that makes sure your Javascript client is a good citizen of your distributed system.",
   "main": "dist/index.js",
   "scripts": {
-    "clean": "rm -rf ./dist",
-    "build": "npm run clean && tsc",
+    "//": "Typescript Compilation",
+
+    "//": "`./testing` files are in typescript for linting, ",
+    "//": "`build-conf`: compiles them for running with karma.",
+
+    "//": "the `|| true` is so that typescript doesn't",
```

Comment:
TypeScript's checks need to fail the build — they usually mean something is actually broken. Was there something specific that was failing unexpectedly?